src/commands/wizard.ts

```diff
@@ -49,8 +41,16 @@ export enum Network {
     mainnet = 'mainnet',
     testnet = 'testnet',
     localNetwork = 'localNetwork',
+    customNetwork = 'customNetwork',
 }
 
+export const assemblies: Record<Network, Assembly[]> = {
+    [Network.mainnet]: [Assembly.dual, Assembly.peer, Assembly.api],
+    [Network.testnet]: [Assembly.dual, Assembly.peer, Assembly.api, Assembly.demo],
+    [Network.localNetwork]: [Assembly.multinode, Assembly.dual, Assembly.peer, Assembly.api, Assembly.demo],
+    [Network.customNetwork]: [Assembly.dual, Assembly.peer, Assembly.api],
```

Comment:
not sure why we need mainnet|testnet|customNetwork, what is _not_ stored in assembly/preset files at this point. 

additionally, should probably use consistent naming mainnet/testnet/local(net?)/custom(net?)

Reply:
`customNetwork` is when a third-party user wants to connect to a custom network (created with symbol-network). The network admin should have shared the custom network.yml preset and nemesis-seed folder. This is just adding the option on the wizard.

About the naming, I'm keen to split the local network's bootstrap preset into -p singleCurrency and -p dualCurrency (current -p bootstrap). What do you think? 

`symbol-bootstrap -p singleCurrency` would be an agnostic local network with just one currency.

Reply:
i'd probably drop testnet preset too since we expect multiple testnets in future it might be confusing

Reply:
Related to the https://github.com/symbol/symbol-bootstrap/pull/341 discussion. 
I would prefer to keep joeynet or any other "commonly" used testnet to simplify the user experience. Other uncommon or experimental testnet can be shared by providing the custom network.yml and seed folder, this PR allows it.

Reply:
we can discuss that more in discord